test/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/Generators.scala

```diff
@@ -43,8 +44,9 @@ object Generators
     with NameMatchGen
     with OnboardingDetailsGen
     with EmailGen
+    with VerifierMatchGen
     with UserTypeGen
-    with VerifierMatchGen {
+    with TriageQuestionsGen {
```

Comment:
By Triage, do we mean ascertaining the eligibility to use the service?

Reply:
basically yeah - there'll be a bunch of exit points in the triage section where the user will have to do something else based on their situation